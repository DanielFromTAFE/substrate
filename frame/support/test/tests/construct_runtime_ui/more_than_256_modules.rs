use frame_support::construct_runtime;

construct_runtime! {
	pub enum Runtime where
		UncheckedExtrinsic = UncheckedExtrinsic,
		Block = Block,
		NodeBlock = Block,
	{
		System: system::{},
		Pallet1: pallet1::{},
		Pallet2: pallet2::{},
		Pallet3: pallet3::{},
		Pallet4: pallet4::{},
		Pallet5: pallet5::{},
		Pallet6: pallet6::{},
		Pallet7: pallet7::{},
		Pallet8: pallet8::{},
		Pallet9: pallet9::{},
		Pallet10: pallet10::{},
		Pallet11: pallet11::{},
		Pallet12: pallet12::{},
		Pallet13: pallet13::{},
		Pallet14: pallet14::{},
		Pallet15: pallet15::{},
		Pallet16: pallet16::{},
		Pallet17: pallet17::{},
		Pallet18: pallet18::{},
		Pallet19: pallet19::{},
		Pallet20: pallet20::{},
		Pallet21: pallet21::{},
		Pallet22: pallet22::{},
		Pallet23: pallet23::{},
		Pallet24: pallet24::{},
		Pallet25: pallet25::{},
		Pallet26: pallet26::{},
		Pallet27: pallet27::{},
		Pallet28: pallet28::{},
		Pallet29: pallet29::{},
		Pallet30: pallet30::{},
		Pallet31: pallet31::{},
		Pallet32: pallet32::{},
		Pallet33: pallet33::{},
		Pallet34: pallet34::{},
		Pallet35: pallet35::{},
		Pallet36: pallet36::{},
		Pallet37: pallet37::{},
		Pallet38: pallet38::{},
		Pallet39: pallet39::{},
		Pallet40: pallet40::{},
		Pallet41: pallet41::{},
		Pallet42: pallet42::{},
		Pallet43: pallet43::{},
		Pallet44: pallet44::{},
		Pallet45: pallet45::{},
		Pallet46: pallet46::{},
		Pallet47: pallet47::{},
		Pallet48: pallet48::{},
		Pallet49: pallet49::{},
		Pallet50: pallet50::{},
		Pallet51: pallet51::{},
		Pallet52: pallet52::{},
		Pallet53: pallet53::{},
		Pallet54: pallet54::{},
		Pallet55: pallet55::{},
		Pallet56: pallet56::{},
		Pallet57: pallet57::{},
		Pallet58: pallet58::{},
		Pallet59: pallet59::{},
		Pallet60: pallet60::{},
		Pallet61: pallet61::{},
		Pallet62: pallet62::{},
		Pallet63: pallet63::{},
		Pallet64: pallet64::{},
		Pallet65: pallet65::{},
		Pallet66: pallet66::{},
		Pallet67: pallet67::{},
		Pallet68: pallet68::{},
		Pallet69: pallet69::{},
		Pallet70: pallet70::{},
		Pallet71: pallet71::{},
		Pallet72: pallet72::{},
		Pallet73: pallet73::{},
		Pallet74: pallet74::{},
		Pallet75: pallet75::{},
		Pallet76: pallet76::{},
		Pallet77: pallet77::{},
		Pallet78: pallet78::{},
		Pallet79: pallet79::{},
		Pallet80: pallet80::{},
		Pallet81: pallet81::{},
		Pallet82: pallet82::{},
		Pallet83: pallet83::{},
		Pallet84: pallet84::{},
		Pallet85: pallet85::{},
		Pallet86: pallet86::{},
		Pallet87: pallet87::{},
		Pallet88: pallet88::{},
		Pallet89: pallet89::{},
		Pallet90: pallet90::{},
		Pallet91: pallet91::{},
		Pallet92: pallet92::{},
		Pallet93: pallet93::{},
		Pallet94: pallet94::{},
		Pallet95: pallet95::{},
		Pallet96: pallet96::{},
		Pallet97: pallet97::{},
		Pallet98: pallet98::{},
		Pallet99: pallet99::{},
		Pallet100: pallet100::{},
		Pallet101: pallet101::{},
		Pallet102: pallet102::{},
		Pallet103: pallet103::{},
		Pallet104: pallet104::{},
		Pallet105: pallet105::{},
		Pallet106: pallet106::{},
		Pallet107: pallet107::{},
		Pallet108: pallet108::{},
		Pallet109: pallet109::{},
		Pallet110: pallet110::{},
		Pallet111: pallet111::{},
		Pallet112: pallet112::{},
		Pallet113: pallet113::{},
		Pallet114: pallet114::{},
		Pallet115: pallet115::{},
		Pallet116: pallet116::{},
		Pallet117: pallet117::{},
		Pallet118: pallet118::{},
		Pallet119: pallet119::{},
		Pallet120: pallet120::{},
		Pallet121: pallet121::{},
		Pallet122: pallet122::{},
		Pallet123: pallet123::{},
		Pallet124: pallet124::{},
		Pallet125: pallet125::{},
		Pallet126: pallet126::{},
		Pallet127: pallet127::{},
		Pallet128: pallet128::{},
		Pallet129: pallet129::{},
		Pallet130: pallet130::{},
		Pallet131: pallet131::{},
		Pallet132: pallet132::{},
		Pallet133: pallet133::{},
		Pallet134: pallet134::{},
		Pallet135: pallet135::{},
		Pallet136: pallet136::{},
		Pallet137: pallet137::{},
		Pallet138: pallet138::{},
		Pallet139: pallet139::{},
		Pallet140: pallet140::{},
		Pallet141: pallet141::{},
		Pallet142: pallet142::{},
		Pallet143: pallet143::{},
		Pallet144: pallet144::{},
		Pallet145: pallet145::{},
		Pallet146: pallet146::{},
		Pallet147: pallet147::{},
		Pallet148: pallet148::{},
		Pallet149: pallet149::{},
		Pallet150: pallet150::{},
		Pallet151: pallet151::{},
		Pallet152: pallet152::{},
		Pallet153: pallet153::{},
		Pallet154: pallet154::{},
		Pallet155: pallet155::{},
		Pallet156: pallet156::{},
		Pallet157: pallet157::{},
		Pallet158: pallet158::{},
		Pallet159: pallet159::{},
		Pallet160: pallet160::{},
		Pallet161: pallet161::{},
		Pallet162: pallet162::{},
		Pallet163: pallet163::{},
		Pallet164: pallet164::{},
		Pallet165: pallet165::{},
		Pallet166: pallet166::{},
		Pallet167: pallet167::{},
		Pallet168: pallet168::{},
		Pallet169: pallet169::{},
		Pallet170: pallet170::{},
		Pallet171: pallet171::{},
		Pallet172: pallet172::{},
		Pallet173: pallet173::{},
		Pallet174: pallet174::{},
		Pallet175: pallet175::{},
		Pallet176: pallet176::{},
		Pallet177: pallet177::{},
		Pallet178: pallet178::{},
		Pallet179: pallet179::{},
		Pallet180: pallet180::{},
		Pallet181: pallet181::{},
		Pallet182: pallet182::{},
		Pallet183: pallet183::{},
		Pallet184: pallet184::{},
		Pallet185: pallet185::{},
		Pallet186: pallet186::{},
		Pallet187: pallet187::{},
		Pallet188: pallet188::{},
		Pallet189: pallet189::{},
		Pallet190: pallet190::{},
		Pallet191: pallet191::{},
		Pallet192: pallet192::{},
		Pallet193: pallet193::{},
		Pallet194: pallet194::{},
		Pallet195: pallet195::{},
		Pallet196: pallet196::{},
		Pallet197: pallet197::{},
		Pallet198: pallet198::{},
		Pallet199: pallet199::{},
		Pallet200: pallet200::{},
		Pallet201: pallet201::{},
		Pallet202: pallet202::{},
		Pallet203: pallet203::{},
		Pallet204: pallet204::{},
		Pallet205: pallet205::{},
		Pallet206: pallet206::{},
		Pallet207: pallet207::{},
		Pallet208: pallet208::{},
		Pallet209: pallet209::{},
		Pallet210: pallet210::{},
		Pallet211: pallet211::{},
		Pallet212: pallet212::{},
		Pallet213: pallet213::{},
		Pallet214: pallet214::{},
		Pallet215: pallet215::{},
		Pallet216: pallet216::{},
		Pallet217: pallet217::{},
		Pallet218: pallet218::{},
		Pallet219: pallet219::{},
		Pallet220: pallet220::{},
		Pallet221: pallet221::{},
		Pallet222: pallet222::{},
		Pallet223: pallet223::{},
		Pallet224: pallet224::{},
		Pallet225: pallet225::{},
		Pallet226: pallet226::{},
		Pallet227: pallet227::{},
		Pallet228: pallet228::{},
		Pallet229: pallet229::{},
		Pallet230: pallet230::{},
		Pallet231: pallet231::{},
		Pallet232: pallet232::{},
		Pallet233: pallet233::{},
		Pallet234: pallet234::{},
		Pallet235: pallet235::{},
		Pallet236: pallet236::{},
		Pallet237: pallet237::{},
		Pallet238: pallet238::{},
		Pallet239: pallet239::{},
		Pallet240: pallet240::{},
		Pallet241: pallet241::{},
		Pallet242: pallet242::{},
		Pallet243: pallet243::{},
		Pallet244: pallet244::{},
		Pallet245: pallet245::{},
		Pallet246: pallet246::{},
		Pallet247: pallet247::{},
		Pallet248: pallet248::{},
		Pallet249: pallet249::{},
		Pallet250: pallet250::{},
		Pallet251: pallet251::{},
		Pallet252: pallet252::{},
		Pallet253: pallet253::{},
		Pallet254: pallet254::{},
		Pallet255: pallet255::{},
		Pallet256: pallet256::{},
	}
}

fn main() {}
