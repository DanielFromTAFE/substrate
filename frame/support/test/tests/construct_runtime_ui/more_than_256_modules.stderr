error: Module index doesn't fit into u8, index is 256
   --> $DIR/more_than_256_modules.rs:265:3
    |
265 |         Pallet256: pallet256::{},
    |         ^^^^^^^^^
